src/vm/lib/metadata/agent.js

```diff
@@ -254,26 +316,35 @@ MetadataAgent.prototype.updateZone = function (zonename, callback) {
 
             old_mtime = (new Date(self.zones[zonename].last_modified));
             if (stats.mtime.getTime() > old_mtime.getTime()) {
-                log.info('last_modified was updated, reloading: ' + zonename);
+                log.info({zonename: zonename},
+                    'last_modified was updated, reloading');
                 cb(true);
                 return;
             }
 
-            log.debug('using cache for: ' + zonename);
+            log.trace('using cache for: ' + zonename);
             cb(false);
         });
     }
 
     shouldLoad(function (load) {
+        var start_lookup_timestamp = (new Date()).getTime();
```

Comment:
It seems we use that value only for debugging, I thought we might have used it for something more critical. I still think it would be an improvement, but I'm not sure that's critical.